tests/integration/test_smoke.py

```diff
@@ -91,6 +91,7 @@
  's3': {'ListBuckets': {}},
  'sdb': {'ListDomains': {}},
  'ses': {'ListIdentities': {}},
+ 'shield': {'GetSubscriptionState': {}},
```

Comment:
I think it may be good to put an explicit integration test where we upper case parts of the endpoint just in-case if `shield` changes their endpoint in the `endpoint.json` to be lowercase.

Reply:
Do you have a suggestion for where to put that?

Reply:
Added a test to `integration/test_client.py`